pom.xml

```diff
@@ -20,6 +20,7 @@
 		<module>org.eclipse.dawnsci.analysis.api</module>
 		<module>../../dawn-third/ncsa.hdf</module>
 		<module>../org.eclipse.richbeans</module>
+		<module>org.eclipse.dawnsci.analysis.asserts</module>
```

Comment:
Well spotted that you have to reference all the modules in the master right now. I think Tycho does actually read the feature but have not checked yet.